tests/test_smoke.py

```diff
@@ -2199,6 +2215,22 @@ def test_azure_disk_tier():
         run_one_test(test)
 
 
+# ------ Testing Zero Quota Failover ------
+@pytest.mark.aws
+def test_aws_zero_quota_failover():
+
+    name = _get_cluster_name()
+    region = get_aws_region_for_quota_failover()
+
+    if not region:
+        return
+
+    test = Test('aws-zero-quota-failover', [
+        f'sky launch -y -c {name} --cloud aws --region {region} --gpus V100:8 --use-spot | grep "Found no quota"',
+    ])
```

Comment:
Although if working normally, this should never create the instance, we still would like to have the termination_cmd to be specified to make sure the instance is cleaned up when leakage happens.
```suggestion
    test = Test(
         'aws-zero-quota-failover', 
         [
            f'sky launch -y -c {name} --cloud aws --region {region} --gpus V100:8 --use-spot | grep "Found no quota"',
         ],
         f'sky down -y {name}'
     )
```